Externals.cfg

```diff
@@ -2,8 +2,8 @@
 protocol = git
 repo_url = https://github.com/ufs-community/regional_workflow
 # Specify either a branch name or a hash but not both.
-#branch = develop
-hash = e5d5c38
+branch = develop
+#hash = e5d5c38
 local_path = regional_workflow
```

Comment:
I assume a hash update commit will be pushed here once the corresponding regional_workflow [PR](https://github.com/ufs-community/regional_workflow/pull/686) is merged?

Reply:
Updated the regional workflow hash.